ts/lib/shortcuts.ts

```diff
@@ -0,0 +1,149 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+import * as tr from "./i18n";
+
+export type Modifier = "Control" | "Alt" | "Shift" | "Meta";
+
+const modifiers: Modifier[] = ["Control", "Alt", "Shift", "Meta"];
+
+// how modifiers are mapped
+const platformModifiers =
+    navigator.platform === "MacIntel"
```

Comment:
When Qt gains native Arm support, I wonder whether this will still be MacIntel?

Reply:
AFAIK a lot of 64-bit Windows still have `Win32` as their platform, so I don't think it's unlikely. :)